explainer.md

```diff
@@ -214,9 +214,12 @@ drawing.removeStroke(stroke)
 // Get a new prediction.
 await drawing.getPrediction()
 
-// Complete the drawing and free up resources. Subsequent calls on the drawing
-// object will throw an error.
-await drawing.finish()
+// Clear all the strokes.
+drawing.clear()
+
+// Complete the recognition and free up resources. Subsequent calls on the 
```

Comment:
The comments needs updating.

---

Out of curiosity, how much resource would a recognizer use (memory)? How about relying on JS garbage collection to determine when to free resources?

```JavaScript
let recognizer = createRecognizer()
...

// Some point after this line, GC will free recognizer object.
recognizer = null
```